samples/bluetooth/rpc_host/sample.yaml

```
@@ -0,0 +1,9 @@
+sample:
+  name: nRF RPC host for BLE API
+  description: nRF RPC host providing BLE API access
+tests:
+  samples.bluetooth.rpc_host:
+    build_only: true
+    build_on_all: true
+    platform_allow: nrf5340dk_nrf5340_cpunet nrf5340pdk_nrf5340_cpunet
```

pdk is no longer officially supported:
```suggestion
    platform_allow: nrf5340dk_nrf5340_cpunet
```